cfme/tests/configure/test_access_control.py

```diff
@@ -21,6 +21,7 @@
 
 records_table = Table("//div[@id='main_div']//table")
 usergrp = ac.Group(description='EvmGroup-user')
+group_table = Table("//div[@id='main_div']//table")
```

Comment:
Just checking that we need to have two separate ```Table``` instances, though I'm guessing that's the reason for the PR.